### PR TITLE
hipcc defaults to code object v3

### PR DESCRIPTION
**Pull request by mangupta**

### CODE REVIEW

File: bin/hipcc [hunk 1]
```diff
@@ -456,6 +457,22 @@ foreach $arg (@ARGV)
         $swallowArg = 1;
     }
 
+    # code object format parsing
+    if ($trimarg eq '-m-code-object-v3') {
```
Comment (yxsamliu) [inline bin/hipcc]: typo

File: bin/hipcc [hunk 2]
```diff
@@ -456,6 +457,22 @@ foreach $arg (@ARGV)
...
+        $coFormatv3 = 1;
+        # hip-clang already recognizes -m-code-object-v3, so we just pass it on
```
Comment (emankov) [inline bin/hipcc]: Сertainly, Sam, should be `-mcode-object-v3`.

**Comment by mangupta** [comment]
@yxsamliu Fixed the typo.

**Comment by yxsamliu** [comment]
> @yxsamliu Fixed the typo.

LGTM. However I do not know if hcc is ready for this, so I will leave to @scchan to decide.

**Pull request merged**